Formula/f/faircamp.rb

```diff
@@ -0,0 +1,49 @@
+class Faircamp < Formula
+  desc "Static site generator for audio producers"
+  homepage "https://codeberg.org/simonrepp/faircamp"
+  url "https://codeberg.org/simonrepp/faircamp/archive/0.9.2.tar.gz"
+  sha256 "15c826e22d8297223ae6c591d14ff79226682ef4db3911a1f349d639406295b7"
+  license "AGPL-3.0-or-later"
+
+  depends_on "opus" => :build
+  depends_on "pkg-config" => :build
+  depends_on "rust" => :build
+  depends_on "ffmpeg"
+  depends_on :macos  # Issues on Linux with brew's libopus, TBD
```

Comment:
can you add a comment about the upstream issue for the linux support?

Reply:
Updated, does this clarify the situation?